File: server/vroom/src/main/java/org/example/vroom/DTOs/responses/route/GetRouteResponseDTO.java
package org.example.vroom.DTOs.responses.route;

import jakarta.persistence.Column;
import lombok.*;

import java.util.List;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class GetRouteResponseDTO {
    Double startLocationLat;
    Double startLocationLng;
    Double endLocationLat;
    Double endLocationLng;
    String startAddress;
    String endAddress;
    List<PointResponseDTO> stops;
}

File: web/vroom/src/app/core/models/ride/responses/get-route-response.dto.ts
import { PointResponseDTO } from "../../driver/point-response.dto";

export interface GetRouteResponseDTO {
  startLocationLat: number;
  startLocationLng: number;
  endLocationLat: number;
  endLocationLng: number;
  startAddress: string;
  endAddress: string;
  stops: PointResponseDTO[];
}